crates/air-lib/interpreter-signatures/src/lib.rs

```diff
@@ -168,9 +161,13 @@ impl From<KeyPair> for fluence_keypair::KeyPair {
 }
 
 pub(crate) fn validate_with_key_format<V>(inner: V, key_format: KeyFormat) -> Result<V, KeyError> {
-    match key_format {
-        fluence_keypair::KeyFormat::Ed25519 => Ok(inner),
-        _ => Err(KeyError::AlgorithmNotWhitelisted(key_format)),
+    // this is needed to satisfy forked fluence-keypar crate which contains only single
+    // variant of KeyFormat, this crate should be refactored in the future
```

Comment:
But new algorithm can be added instead.
Or is it intended that fluence_keypair will contain only whilelisted algos?

Reply:
It seems to work with `#[allow(unreachable_patterns)]` enabled locally.

Reply:
Fixed